### PR TITLE
feat: Add "No credentials yet" screen

### LINKED ISSUE #972: Add "No credentials yet" screen
https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/latest/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/DD640204-60A3-4DA4-88D9-6067069824BB?collectionId=ba692773-75df-473a-a41c-38a0647809a4&collectionLayerId=73f3e013-9f94-4b90-bfd2-f35c4f6b005a

### PR DESCRIPTION
**Pull request by talwinder50**
svgs are optimized 

Screen Shot 2021-08-20 at 1 13 15 PM
closes #972

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>
Screen Shot 2021-08-20 at 3 00 38 PM

### CODE REVIEW

File: cmd/wallet-web/src/pages/Dashboard.vue [hunk 1]
```diff
@@ -60,6 +82,7 @@ import { CredentialManager } from '@trustbloc/wallet-sdk';
 import { getCredentialType } from '@/pages/mixins';
 import { mapGetters } from 'vuex';
 import SkeletonLoader from '../components/SkeletonLoader/SkeletonLoader';
+import FlyoutMenu from '../components/FlyoutMenu/FlyoutMenu';
 import credentialDisplayData from '@/config/credentialDisplayData.json';
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Dashboard.vue:86]: Let's use `@/components` instead

File: cmd/wallet-web/src/pages/Dashboard.vue [hunk 2]
```diff
@@ -5,15 +5,29 @@
...
+        <div
+          class="
+            inline-flex
+            items-center
+            bg-neutrals-white
+            rounded-lg
+            border border-neutrals-chatelle
+          "
+        >
+          <flyout-menu></flyout-menu>
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Dashboard.vue:24]: `<flyout-menu />`

File: cmd/wallet-web/src/pages/Dashboard.vue [hunk 3]
```diff
@@ -69,15 +92,16 @@ export default {
...
   data() {
     return {
       cards: [],
       cjson: credentialDisplayData,
       username: '',
       agent: null,
-      error: 'No stored credentials',
-      errorDescription: "Your wallet is empty, there aren't any stored credentials to show.",
+      error: 'No credentials yet',
+      errorDescription: 'Credentials you add will appear here',
```
Comment (sudeshrshetty) [inline cmd/wallet-web/src/pages/Dashboard.vue:104]: Should be read from i18n resource bundle?

Comment (sudeshrshetty) [reply cmd/wallet-web/src/pages/Dashboard.vue:104]: @birtony ^^

Comment (birtony) [reply cmd/wallet-web/src/pages/Dashboard.vue:104]: If we have that in designs

Comment (talwinder50) [reply cmd/wallet-web/src/pages/Dashboard.vue]: @sudeshrshetty done

**Review by birtony — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-web/src/translations/en.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ import DashboardLayout from './DashboardLayout/en.json';
 import Signin from './Signin/en.json';
 import Signout from './Signout/en.json';
 import Signup from './Signup/en.json';
+import Credentials from './Credentials/en.json';
```
Comment (sudeshrshetty) [inline cmd/wallet-web/src/translations/en.js:7]: translation file name is too generic, how about credential view, share, save? something to revisit later.

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/wallet-web/src/translations/en.js (hunk 4 above)
Comment (rolsonquadras) [reply cmd/wallet-web/src/translations/en.js:7]: @talwinder50 ^

**Pull request merged**